hawc/apps/summary/serializers.py

```diff
@@ -53,6 +54,8 @@ class Meta:
 
 
 class VisualSerializer(CollectionVisualSerializer):
+    rob_settings = AssessmentRiskOfBiasSerializer(source="assessment")
```

Comment:
This is the root cause of all the additions. should we make this a conditional to only include this if we're dealing with  `if self.visual_type in [self.ROB_HEATMAP, self.ROB_BARCHART]`, else null?